### ISSUE TITLE
Permission denied on Deb installation on Debian or Ubuntu

### ISSUE THREAD

**Issue by ricardocassiano04** [adoptium/installer#622]
Hello!

Unlike **apt** (step 1) command,  **mkdir** (step 2)  and **tee** (steps 2 and 3) commands will not ask por sudo password.

So the repository will not be configured.

https://adoptium.net/installation/linux/

Contratulations for your good job!

**Comment by karianna** [comment]
@ricardocassiano04 So you are saying that `mdir` and `tee` _need_ to be run as sudo?

**Comment by ricardocassiano04** [comment]
Yes.

**Comment by zdtsw** [comment]
I think in the "Tip" it already states :
"On most Linux systems you must have superuser privileges to install packages such as Temurin. You may need to prefix the commands below with sudo for them to succeed."

**Comment by sxa** [comment]
> So you are saying that mdir and tee need to be run as sudo?

Yeah the `tee` is the bit that writes to files in `/etc` so needs to be run as root.

> Unlike apt (step 1) command,

@ricardocassiano04 I'm curious - are you saying the the `apt install` will prompt for the password if it's run as non-root? That doesn't usually happen in my experience - which distribution+version is that happening on?

On Ubuntu 22.04 it doesn't seem to try to prompt me and just fails:
```
$ apt install -y wget apt-transport-https
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?
$ 
```
